src/jit/emitarm64.cpp

```diff
@@ -6784,7 +6796,77 @@ void emitter::emitIns_R_D(instruction ins, emitAttr attr, unsigned offs, regNumb
 
 void emitter::emitIns_J_R(instruction ins, emitAttr attr, BasicBlock* dst, regNumber reg)
 {
-    NYI("emitIns_J_R");
+    assert((ins == INS_cbz) || (ins == INS_cbnz));
+
+    assert(dst && (dst->bbFlags & BBF_JMP_TARGET));
```

Comment:
nit: coding convention: compare against nullptr. Also, please separate the asserts to it's easier to see which one failed:
```
assert(dst != nullptr);
assert((dst->bbFlags & BBF_JMP_TARGET) != 0);
```
(below, also)